packages/rs-dapi-client/src/request_settings.rs

```diff
@@ -64,12 +65,13 @@ impl RequestSettings {
             ban_failed_address: self
                 .ban_failed_address
                 .unwrap_or(DEFAULT_BAN_FAILED_ADDRESS),
+            ca_certificate: None,
         }
     }
 }
 
 /// DAPI settings ready to use.
-#[derive(Debug, Clone, Copy)]
+#[derive(Debug, Clone)]
 pub struct AppliedRequestSettings {
```

Comment:
So we override CA certificate for each request? Does it make sense? I feel like it must be only in global settings not?